packages/ssi-service/src/feedback-panel/api.js

```diff
@@ -1,67 +1,49 @@
 /**
- * Helper class for making API calls to the One Platform API Gateway
+ * MIT License
+ * Copyright (c) 2021 Red Hat One Platform
+ * 
+ * @version 0.1.0
+ * 
+ * This is the api interface for the opc-feedback component. This supports passing the payload of 
+ * feedback data that needs to be passed to backend.
+ * @params - feedbackInput: Feedback data which needs to be passed to backend
+ * 
+ * @summary feedback.js enables to to interact with feedback microservice. 
+ * @author Rigin Oommen <riginoommen@gmail.com>
+ *
+ * Created at     : 2021-01-14 13:50:01
+ * Last modified  : 2021-03-01 12:40:27
  */
-export class APIService {
-  constructor () {
-    this._apiBasePath = process.env.APPS_BASE_API;
-  }
 
-  /**
-   * Returns the request headers with Authorization if jwtToken is present
-   */
-  get _headers () {
-    const headers = {
-      'Content-Type': 'application/json',
-    }
-
-    if ( this.authToken ) {
-      headers.Authorization = 'Bearer ' + this.authToken;
-    } else if ( window.OpAuthHelper?.jwtToken ) {
-      headers.Authorization = 'Bearer ' + window.OpAuthHelper.jwtToken;
-    }
-
-    return headers;
-  }
-
-  /**
-   * Use a custom endpoint or auth token for the Feedback Graphql APIs
-   * @param {{ endpoint: string, authToken: string }} basePath
-   */
-  config ( { endpoint, authToken } ) {
-    if ( endpoint ) {
-      this._apiBasePath = endpoint;
-    }
-    if ( authToken ) {
-      this.authToken = authToken;
-    }
-  }
-
-  /**
-   * Sends a graphql request
-   */
-  request ( query, variables = undefined ) {
-    return fetch( this._apiBasePath, {
-      method: 'POST',
-      headers: this._headers,
-      body: JSON.stringify( {
-        query,
-        variables,
-      } ),
-    } )
-      .then( res => {
-        if ( !res ) {
-          throw new Error( 'The Server returned an empty response' );
+sendFeedback = (feedbackInput) => {
+    let feedbackQuery = JSON.stringify({
+        query: `mutation CreateFeedback($input: FeedbackInput!) {
+        createFeedback(input: $input) {
+            _id
+            ticketUrl
+            }
+        }`,
+        variables: {
+            "input": feedbackInput
         }
-        return res.json();
-      } )
-      .then( res => {
-        if ( res.errors ) {
-          throw new Error( 'There were some errors in the query' + JSON.stringify( res.errors ) );
+    });
+    let xhr = new XMLHttpRequest();
```

Comment:
Can you replace this with a simple javascript fetch function?

Lookup [javascript fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch), xhr is very outdated and you don't have to construct an xhr object just to send a simple POST request.